hack/install-kubevirtci.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 organization=kubevirt
-commit="6132f2efcf2aa2d48f256a409fe90cc0b74c3563"
+commit="78dcc4f8aaec467bba8ae325646056128268a169"
```

Comment:
do we want to take bump-kubevirtci from kubevirt to k8s-nmstate ?

Reply:
It does not make that much sense here. We don't keep kubevirtci in the source tree, all needed here is to change this single hash. And I'd rather do it manually than keep a script for one-line change.